segments-guide.md

```diff
@@ -0,0 +1,327 @@
+# Segments APIs
+
+## Segments Overview
+
+The Analytics 2.0 Segments APIs allow you to retrieve, update, or create segments programmatically through Adobe I/O. The APIs use the same data and methods that are used when working with segments in the UI. For more information on Adobe Analytics segments, see [Adobe Help](https://marketing.adobe.com/resources/help/en_US/analytics/segment/seg_overview.html). For more information on understanding segment definition data architecture, see the [Segment Definition](segments.md) article that accompanies this user guide.
+
+## Authorization and Authentication
+
+To obtain authorization and authentication, see the [Getting Started Guide for 2.0 APIs](https://www.adobe.io/apis/experiencecloud/analytics/docs.html#!AdobeDocs/analytics-2.0-apis/master/create-oauth-client.md).
+
+## Best Practices
+
+Please follow these guidelines when using the Segments APIs:
+
+*  Make multiple, smaller requests instead of a large, single request.
+*  Request data once and cache it.
+*  Use caution when updating a segment so that you do not alter the original segment used by others.
+*  Avoid creating duplicate segments with the same definition. Creating many segments will affect performance for your company in some situations.
+*  Use the Analytics UI for creating segments from scratch, if possible. Because of the flexibility in using segments, the UI offers significant advantages for managing and optimizing the complexity of this task. Consider creating a segment in the UI that matches, as closely as possible, the segment you want, then modifying the JSON so that it does what you want programmatically. Then you can `POST` the JSON body of that modified segment to the API.
```

Comment:
This paragraph is a bit awkward. You may want to reword it. Below is an attempt at streamlining it a bit. 

"If possible use the Analytics UI for creating segments from scratch. The UI offers significant advantages for managing and optimizing the complexity of this task. If you need to create segments programmatically it is usually easier to create a template segment in the UI and then have your application only change small portions of the segment definition."